components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -181,4 +182,24 @@ public void testBadArgument() throws FormatException, IOException {
         System.getProperty("line.separator"), outContent.toString());
     }
   }
+
+  @Test
+  public void testCompanion() throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tif");
+    outFile.delete();
+    File compFile = new File(outFile.getParentFile(), "test.companion.ome");
+    String[] args = {
+      "-option", OMETiffWriter.COMPANION_KEY, compFile.getAbsolutePath(),
+      "test.fake", outFile.getAbsolutePath()
+    };
+    try {
+      ImageConverter.main(args);
```

Comment:
Yes very similarly to the discussion we had in https://github.com/openmicroscopy/bioformats/pull/2737#discussion_r98410681. I think the alternatives are:
- pass `-overwrite` to the ImageConverter command
- move to a strategy similar to  `OMETiffWriterTest` i.e. create a temp directory, resolve the path without creating the unique file

Thoughts?